meshery-design-embed/src/App.jsx

```diff
@@ -8,22 +8,24 @@ function App() {
   // By default, this embed points to https://meshery.layer5.io.
   // To change the destination, modify the host or designId below.
 
```

Comment:
Why are we including a link to an actual design? Please change this to a placeholder that users will need to fill-in with their design ID.

```
https://cloud.layer5.io/catalog/content/design/<your-design-id-here>
```